Doc/library/time.rst

```diff
@@ -388,6 +388,22 @@ Functions
       by a signal, except if the signal handler raises an exception (see
       :pep:`475` for the rationale).
 
+.. function:: sleep_until(secs)
+
+   Like :func:`sleep`, but sleep until the specified time of the system clock (as
+   returned by :func:`time`). This can be used, for example, to schedule events
```

Comment:
I'm not convinced from this documentation what is the benefit of using this function instead of existing code:

```
dt = deadline - time.time()
if dt > 0: time.sleep(dt)
```

How does it behave differently? If it does behave the same, the function has no benefit?